kuma/core/templatetags/jinja_helpers.py

```diff
@@ -187,3 +188,8 @@ def get_locale_localized(locale):
     language = settings.LOCALES[locale].english
 
     return _(language)
+
+
+@library.filter
+def money(value, currency="USD"):
+    return format_currency(value, currency)
```

Comment:
So this doesn't respect the used locale yet. But it could be extended to do that, once we go multi-national!

Reply:
One thing at a time I guess. Having this here is powerful because it'll be just one single point for ruling all formatting. 
That `format_currency` does take a `locale` option but we can worry about that some other day.